sn_networking/src/lib.rs

```diff
@@ -387,11 +386,12 @@ impl SwarmDriver {
             pending_record_put: Default::default(),
             replication_fetcher: Default::default(),
             local,
-            // We use 63 here, as in practice the capactiy will be rounded to the nearest 2^(n-1).
+            // We use 63 here, as in practice the capacity will be rounded to the nearest 2^(n-1).
```

Comment:
Noticed there's another typo here: should be ` 2^n-1` instead of `2^(n-1)`
63 made no sense if it was `2^(n-1)`, checked with the link below, it should indeed be `2^n-1`